addons/website/data/website_data.xml

```diff
@@ -498,37 +498,37 @@
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_1.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/furniture.png</field>
         </record>
         <record id="website.s_product_list_default_image_2" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_2.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/clothes.jpeg</field>
         </record>
         <record id="website.s_product_list_default_image_3" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_3.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/books.jpeg</field>
         </record>
         <record id="website.s_product_list_default_image_4" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_4.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/bricolage.jpg</field>
         </record>
         <record id="website.s_product_list_default_image_5" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_5.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/service.png</field>
         </record>
         <record id="website.s_product_list_default_image_6" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_6.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/multimedia.png</field>
```

Comment:
@sbuhl Convinced by this change for images of the items snippet? Seems like random images and they are not even using the same ratios... :grimacing:

Reply:
I'll find other images.